src/core_ocean/shared/mpas_ocn_gm.F

```diff
@@ -219,7 +221,13 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, diagnosticsPool, &
 
       allocate(gradDensityEdge(nVertLevels, nEdges), &
                dzdxEdge(nVertLevels, nEdges), &
-               drhoDzTopOfEdge(nVertLevels, nEdges))
+               drhoDzTopOfEdge(nVertLevels, nEdges), &
+              dDensityDzTopOfCell(nVertLevels, nCells), &
```

Comment:
Some of these new arrays need nVertLevels+1. I can fix that.

Reply:
okay, thanks!  I could also take a look and fix as well if you'd like